pytorch/inference/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -142,14 +142,14 @@ RUN pip uninstall -y torch torchvision torchaudio torchdata model-archiver multi
 RUN pip install --no-cache-dir --extra-index-url https://download.pytorch.org/whl/cpu -U \
     "awscli<2" \
     boto3 \
-    "cryptography>3.2" \
+    cryptography \
```

Comment:
why remove this lower bound pin?

Reply:
cryptography pin is way to low as the package changed from 3.4.8 to 35.0.0 and is currently up to 39.0.1. As building the image will already use latest (in most cases), there is no need to maintain the pinning here, unless it is not getting a high enough version.